ouroboros-consensus-diffusion/src/diffusion-testlib/Test/ThreadNet/Network.hs

```diff
@@ -599,13 +597,16 @@ runThreadNetwork systemTime ThreadNetworkArgs
       -> (SlotNo -> STM m ())
       -> LedgerConfig blk
       -> STM m (Point blk)
-      -> (ResourceRegistry m -> m (DiskLedgerView m (ExtLedgerState blk)))
+      -> m ( LedgerDB' blk
+           , LedgerBackingStoreValueHandle m (ExtLedgerState blk)
+           , DiskLedgerView m (ExtLedgerState blk)
+           )
       -> Mempool m blk
       -> [GenTx blk]
          -- ^ valid transactions the node should immediately propagate
       -> m ()
     forkCrucialTxs clock s0 registry unblockForge lcfg getTipPoint mdlv mempool txs0 = do
-      void $ forkLinkedThread registry "crucialTxs" $ withRegistry $ \reg -> do
+      void $ forkLinkedThread registry "crucialTxs" $ do
```

Comment:
Should we still do something with the registry here?

Reply:
Nope